dotnet/samples/Concepts/Memory/VectorStore_VectorSearch_MultiVector.cs

```diff
@@ -0,0 +1,144 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Azure.Identity;
+using Microsoft.SemanticKernel.Connectors.AzureOpenAI;
+using Microsoft.SemanticKernel.Data;
+using Microsoft.SemanticKernel.Embeddings;
+
+namespace Memory;
+
+/// <summary>
+/// An example showing how to do vector search where there may be multiple vectors
+/// stored in each record and you want to specify which vector to search on.
+///
+/// The example shows the following steps:
+/// 1. Create a Volatile Vector Store.
+/// 2. Generate and add some test data entries.
+/// 3. Search for records based on a specified vector.
+/// </summary>
+public class VectorStore_VectorSearch_MultiVector(ITestOutputHelper output) : BaseTest(output)
+{
+    [Fact]
+    public async Task ExampleAsync()
```

Comment:
Prefer longer for descriptive method names as this will help if we add another Fact to this class